tests/unit/synapseclient/unit_test_tables.py

```diff
@@ -171,6 +172,17 @@ def test_materialized_view():
     assert mat_view.properties.columnIds == []
 
 
+def test_dataset():
+    dataset = Dataset(
```

Comment:
Do we need to test what happens when `items` is passed in?

Reply:
Agreed. The tests definitely need to be expanded.